types/spectest/tests/ssz/ssz_spectest.go

```diff
@@ -0,0 +1,41 @@
+package ssz
+
+import (
+	"testing"
+
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/stretchr/testify/require"
+
+	"github.com/bloxapp/ssv-spec/types"
+)
+
+type SSZSpecTest struct {
+	Name          string
+	Data          []byte
+	ExpectedRoot  [32]byte
+	ExpectedError string
+}
+
+func (test *SSZSpecTest) TestName() string {
+	return test.Name
+}
+
+func (test *SSZSpecTest) Run(t *testing.T) {
+	cd := &types.ConsensusData{}
+	require.NoError(t, cd.Decode(test.Data))
+
+	vBlk, _, err := cd.GetBlockData()
+	require.NoError(t, err)
+	require.NotNil(t, vBlk)
+
+	var withdrawals types.SSZWithdrawals
```

Comment:
As discussed with Oleg, currently SSZSpecTest is only good for withdrawals but later we can come up with better infra

Reply:
would love a brief explanation about this whole folder `ssz`, so we're testing that we're able to serialize and de-serialize our objects using SSZ?

Reply:
Yes,

Note we don't fully rely on a third party lib, as we copy-pasted from there :-P
See https://github.com/bloxapp/ssv-spec/pull/234#discussion_r1193130165

So we must test